app/models/category.rb

```diff
@@ -1,3 +1,8 @@
 class Category < ActiveRecord::Base
-  has_many :videos, -> { order("title") }
+  has_many :videos, -> { order(created_at: :desc) }
+
+  def recent_videos
+    self.videos.limit(6)
```

Comment:
I use it as a visual reminder that's I'm acting on a instance. Is it in bad taste to explicitly declare it?

Thank you so much for your feedback Albert!